### ISSUE TITLE
use in browser error : Uncaught ReferenceError: exports is not defined

### ISSUE THREAD

**Issue by neozhu** [jeanlescure/short-unique-id#50]
<script src="https://cdn.jsdelivr.net/npm/short-unique-id@latest/dist/short-unique-id.min.js"></script>

    
    <script>
        // Instantiate
        var uid = new ShortUniqueId();

        // Random UUID
        document.write(uid());

        // Sequential UUID
        document.write(uid.seq());
    </script>

![image](https://user-images.githubusercontent.com/1549611/132830268-a375eff3-8afd-4874-84f7-3e36b150f9de.png)

**Comment by jeanlescure** [comment]
We have issued a fix on v4.4.2

As per the link in the README, it points to the `@latest` tag in JSDelivr which is still serving (from cache) the broken v4.4.0

I'm following the procedure to hopefully have this cache purged ASAP.

In the meantime, if this problem is time-sensitive, the aforementioned JSDelivr link can be replaced with the following in order to fix it:

```
https://cdn.jsdelivr.net/npm/short-unique-id@4.4.2/dist/short-unique-id.min.js
```

**Comment by neozhu** [comment]
Thank you very much.  keep safe and have a nice time.
